library/pcs_cluster.py

```diff
@@ -127,6 +141,7 @@ def run_module():
             cluster_name=dict(required=False),
             token=dict(required=False, type='int'),
             transport=dict(required=False, default="default", choices=['default', 'udp', 'udpu', 'knet']),
+            transport_options=dict(required=False, type='str'),
```

Comment:
(medium priority) add `default=''` so we don't need to use the 'if' condition later, default (if not specified) will be empty string